Makefile

```diff
@@ -26,20 +26,40 @@ run: ## Run the wallabag built-in server
 build: ## Run webpack
 	@npm run build:$(ENV)
 
-test: ## Launch wallabag testsuite
-	@ant prepare && bin/simple-phpunit -v
+prepare: clean ## Prepare database for testsuite
+ifdef DB
+	cp app/config/tests/parameters_test.$(DB).yml app/config/parameters_test.yml
+endif
+	-php bin/console doctrine:database:drop --force --env=test
+	php bin/console doctrine:database:create --env=test
+ifndef DB ## make test does not define DB
+	php bin/console doctrine:schema:create --env=test
+endif
+ifeq ($(DB), sqlite)
+	php bin/console doctrine:schema:create --env=test
+endif
+ifeq ($(DB), mysql)
```

Comment:
I wonder if that can not be simplified by creating an initial migration. If there is already migration executed, the migration would be marked migrated via `$this->version->markMigrated();`. So, instead of `php bin/console doctrine:database:import data/sql/mysql_base.sql --env=test` and `psql -h localhost -d wallabag_test -U travis -f data/sql/pgsql_base.sql` coulb replaced by `php bin/console doctrine:migrations:migrate --no-interaction --env=test`


If you are interested, once merged, I could create a PR

Reply:
You don't need to wait for that PR to be merged, you create a brand from `issue-1735` and target that branch for your PR